package.json

```diff
@@ -48,7 +48,8 @@
     "fastify-plugin": "^3.0.0",
     "fastify-static": "^3.3.0",
     "js-yaml": "^3.14.0",
-    "json-schema-resolver": "^1.2.0"
+    "json-schema-resolver": "^1.2.0",
+    "openapi-types": "^7.0.1"
```

Comment:
added new package with ^ as all the other packages. (seems to be the prefered way here)

Reply:
I would not recommend adding this dependency, maybe as a devDep?

Reply:
good finding, already corrected